Makefile

```diff
@@ -22,18 +22,81 @@ REGISTRY ?= gcr.io/heptio-images
 IMAGE = $(REGISTRY)/$(TARGET)
 DOCKER ?= docker
 DIR := ${CURDIR}
-VERSION ?= v0.1
+VERSION ?= v0.2
+
+ARCH    ?= amd64
+LINUX_ARCHS = amd64 arm64
+PLATFORMS = linux/amd64,linux/arm64
+
+IMAGEARCH ?=
+QEMUARCH  ?=
+
+MANIFEST_TOOL_VERSION := v1.0.0-rc
+
+ifneq ($(ARCH),amd64)
+IMAGE_TAG = $(IMAGE_REPO)-$(ARCH):$(IMAGE_TAG_NAME)
```

Comment:
I dont see where `IMAGE_REPO` gets set. 

Also, we dont need the suffix if making amd64? Just checking against what we do in the Sonobuoy repo, we push all the images named as `sonobuoy-$arch` and the manifest is pushed to `sonobuoy`. We do build/tag the image locally as `sonobuoy` though so if you are just building for local use it has the more intuitive name.

Reply:
Hi @johnSchnake,
I have fixed it.
Please check.